runconfig/config_windows.go

```diff
@@ -0,0 +1,13 @@
+package runconfig
+
+// ContainerConfigWrapper is a Config wrapper that hold the container Config (portable)
+// and the corresponding HostConfig (non-portable).
+type ContainerConfigWrapper struct {
+	*Config
+	HostConfig *HostConfig `json:"HostConfig,omitempty"`
```

Comment:
i assume this is not exported via an API, otherwise Linux would call this InnerHostConfig whereas windows calls it HostConfig.